cmd/dendrite-sync-api-server/main.go

```diff
@@ -28,9 +28,11 @@ func main() {
 	accountDB := base.CreateAccountsDB()
 	federation := base.CreateFederationClient()
 
-	_, _, query := base.CreateHTTPRoomserverAPIs()
+	fsAPI := base.CreateHTTPFederationSenderAPIs()
+	rsAPI := base.CreateHTTPRoomserverAPIs()
+	rsAPI.SetFederationSenderAPI(fsAPI)
```

Comment:
I don't understand why we've sprouted an additional dependency on `fsAPI` here. Is this because one of `alias` or `input` APIs required it, and hence to make `rsAPI` we require it? So even though we don't actually invoke those methods, we need to supply them?

Reply:
I think in this instance I was just trying to be consistent but as far as I can tell, none of the sync API calls will call a path that requires the federation sender. I'll remove this.